frontend/app/utils/sort-by-year-old.js

```diff
@@ -0,0 +1,47 @@
+import { computed, get } from "@ember/object";
+
+// remove this helper after completely moving over to the new version
```

Comment:
looks like it would be nice to have some unit tests here. nice task for the students ?